packages/web-config-server/src/apiV1/dataBuilders/generic/latestData/latestDataValue.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Tupaia Config Server
+ * Copyright (c) 2019 Beyond Essential Systems Pty Ltd
+ */
+
+import { DataBuilder } from '/apiV1/dataBuilders/DataBuilder';
+import { AGGREGATION_TYPES } from '@tupaia/aggregator/dist/aggregationTypes';
+
+
+export class LatestDataValueBuilder extends DataBuilder {
+  async build() {
+    const { dataElementCodes } = this.config;
+    const { results } = await this.fetchAnalytics(dataElementCodes);
+    const [ result ] = results;
+    return { value: result?.value ?? null };
+  }
+}
+
+export const latestDataValue = async ({ models, dataBuilderConfig, query, entity }, aggregator, dhisApi) => {
+  const builder = new LatestDataValueBuilder(
+    models,
+    aggregator,
+    dhisApi,
+    dataBuilderConfig,
+    query,
+    entity,
+    AGGREGATION_TYPES.MOST_RECENT,
```

Comment:
```suggestion
    aggregator.aggregationTypes. MOST_RECENT,
```

With this we don't need to import aggreegation types from dist.

Reply:
An example: [ValuesPerPeriodByOrgUnit.js L72](https://github.com/beyondessential/tupaia/blob/857481c74d59f537026e59aa8927269a86ee7b35/packages/web-config-server/src/apiV1/dataBuilders/generic/comparison/valuesPerPeriodByOrgUnit.js#L72)

Reply:
Updated